doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -375,6 +386,19 @@ At this time, we do not make any guarantees about the freshness of these numbers
 | `slowest_local_input_id`  | [`text`]         | The ID of the slowest direct input.                                                      |
 | `slowest_global_input_id` | [`text`]         | The ID of the slowest root input.                                                        |
 
+### `mz_materialized_view_refresh_strategies`
```

Comment:
Looking at it again, would it be clearer to call this `mz_materialized_view_refresh_options`? Maybe that's clearer? Maybe it's not. Thought I'd throw it out though. Not a blocking concern.

Reply:
It only makes sense to use `_strategies` if we consistently refer to this feature as...a materialized view's refresh strategy, as a general rule. @sthm has been the main user of this feature, so he should be a good barometer of what's more intuitive.

Not strongly opposed to going back to `_options`, though! I just picture this feature being documented as "refresh strategies" rather than "refresh options".

Reply:
> Not strongly opposed to going back to `_options`, though! I just picture this feature being documented as "refresh strategies" rather than "refresh options".

I think that's exactly the right way to make the decision! Let's stick with `_strategies` then.

Reply:
I think we have naturally been using strategies so far, so that checks out.

Reply:
Ok, thank you! Going with `_strategies`.